components/Votes/Votes.tsx

```diff
@@ -431,17 +436,21 @@ const Title = styled.h1`
 
 const ButtonWrapper = styled.div`
   display: flex;
-  flex-direction: column;
-  align-items: end;
-  justify-content: center;
+  justify-content: end;
```

Comment:
something happened to this button wrapper which was putting the buttons in a column rather than a row. tried to correct it here, might want to double chekc this work